scripts/e2e-tests.sh

```diff
@@ -93,15 +93,30 @@ function dump_cluster_state() {
   echo "***         E2E TEST FAILED         ***"
   echo "***    Start of information dump    ***"
   echo "***************************************"
-  echo ">>> All resources:"
-  kubectl get all --all-namespaces --show-labels -o wide
-  echo ">>> Services:"
-  kubectl get services --all-namespaces --show-labels -o wide
-  echo ">>> Events:"
-  kubectl get events --all-namespaces --show-labels -o wide
-  function_exists dump_extra_cluster_state && dump_extra_cluster_state
-  echo ">>> Full dump: ${ARTIFACTS}/k8s.dump.txt"
-  kubectl get all --all-namespaces -o yaml > ${ARTIFACTS}/k8s.dump.txt
+
+  local output="${ARTIFACTS}/tmp/k8s.dump.txt"
```

Comment:
I love this idea of not bloating e2e logs!
Don't you need to create the `${ARTIFACTS}/tmp` dir before dumping logs?

Reply:
/tmp is a bug, is is an artifact from my local debugging.
Good catch, will fix.

Reply:
Maybe applying this suggestion is easier? :)

```suggestion
  local output="${ARTIFACTS}/k8s.dump.txt"
```